lib/src/core.dart

```diff
@@ -65,12 +59,8 @@ class AudioSession {
           break;
       }
     });
-    _avAudioSession?.silenceSecondaryAudioHintStream
-        ?.listen((event) => _silenceSecondaryAudioHintSubject.add(event));
-    _avAudioSession?.mediaServicesWereLostStream
-        ?.listen((event) => _mediaServicesWereLostSubject.add(event));
-    _avAudioSession?.mediaServicesWereResetStream
-        ?.listen((event) => _mediaServicesWereResetSubject.add(event));
+    _avAudioSession?.becomingNoisyEventStream
+        ?.listen((event) => _becomingNoisyEventSubject.add(event));
```

Comment:
Added the _becomingNoisyEventSubject listener here and corresponding _becomingNoisyEventSubject in darwin.dart